bean-generator-bv-1.1/src/main/resources/generator.default.properties

```diff
@@ -21,7 +21,7 @@
 # You can override any value by creating a generator.properties file
 #
 generator.rnd.seed=12
-generator.outputdir=./target/generated-sources/beangraph
+generator.outputdir=./bean-generator-bv-1.1/target/generated-sources/beangraph
```

Comment:
So basically, this is the thing we have to change if we want to remove the parent module?